app/ping/judge_endpoint/endpoint_check.py

```diff
@@ -0,0 +1,55 @@
+"""Code that runs checks in order to confirm data that will be sent to Judge is correct"""
+
+from flask import request, json
+from app.middleware.request_validations import Middleware
+
+keys_requested = (
```

Comment:
```suggestion
KEYS_REQUESTED = (
```
Pon el nombre en mayuscula, crea una carpeta utils en el path de app/utils y crea un archivo constants.py
Que quede con ruta app/utils/constants.py y ahí guarda esto